src/languages/en.ts

```diff
@@ -1984,7 +1988,7 @@ export default {
         parentNavigationSummary: ({rootReportName, workspaceName}: ParentNavigationSummaryParams) => `From ${rootReportName}${workspaceName ? ` in ${workspaceName}` : ''}`,
     },
     qrCodes: {
-        copy: 'Copy',
+        copy: 'Copy URL',
```

Comment:
Is this discussed?
Seems like "Copy" was approved by marketing team [here](https://github.com/Expensify/App/issues/32282#issuecomment-1839131487)

Reply:
Taken from [Figma](https://www.figma.com/file/Xv3yHJooZcxRGkXbAcTZD5/%23ideal-nav-(Ideal-Nav)?type=design&node-id=3125-183883&mode=design&t=gvgQ1oUvKrfoGp2M-4) cc @shawnborton @dubielzyk-expensify

Reply:
Also ccing @jamesdeanexpensify

Reply:
Got it, I guess we should go with Copy. Though I feel like "Copy URL" is more appropriate, because that's what actually happens. You don't copy the image, you copy the url (I think). cc @trjExpensify @dannymcclain for thoughts too

Reply:
That's fair. If it just says "Copy" I'd personally expect it to copy the full image somehow, but that's what "Download" does already, so I can see us being a bit more specific there.

Reply:
Cool, let's do Copy URL then. Thanks James!